native/explorer/src/dataframe.rs

```diff
@@ -244,10 +243,23 @@ pub fn df_width(data: ExDataFrame) -> Result<usize, ExplorerError> {
 }
 
 #[rustler::nif(schedule = "DirtyCpu")]
-pub fn df_vstack(data: ExDataFrame, other: ExDataFrame) -> Result<ExDataFrame, ExplorerError> {
+pub fn df_vstack_many(
```

Comment:
On this one, I think it might be more something like:

```rust
#[rustler::nif(schedule = "DirtyCpu")]
pub fn df_vstack(
    data: ExDataFrame,
    others: Vec<ExDataFrame>,
) -> Result<ExDataFrame, ExplorerError> {
    let df = &mut data.resource.0.clone();
    let new_df = others
        .iter()
        .fold(df, |acc, df2| acc.vstack_mut(&df2.resource.0)?);
    Ok(ExDataFrame::new(new_df.clone()))
}
```

(this is untested)

Reply:
Fantastic!